deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml

```
@@ -72,6 +72,50 @@ spec:
                 - parameter
                 type: object
               type: array
+            vault:
```

I would rename this to `hashiCorpVault` so that it's explicit which vault it is, for example it can be confusing with Azure Key Vault how it is now.
```suggestion
            hashiCorpVault:
```

---

done :)

---

Thank you!